returnn/config.py

```diff
@@ -190,7 +190,12 @@ def add_line(self, key, value):
       if value_type == str:
         pass  # keep as-is
       else:
-        value = eval(value)
+        try:
+          value = eval(value)
+        except SyntaxError:
+          from returnn.log import log
+          print("WARNING: can't evaluate config param '%s' to previous type: %s. Keeping as string."
+                % (value, value_type), file=log.v1)
```

Comment:
Do not use a hanging indent like that.
```suggestion
          print(
            "WARNING: can't evaluate config param '%s' to previous type: %s. Keeping as string." % (value, value_type),
            file=log.v1)
```

Reply:
Ok, else the line is over 120 chars, should I change the message?

Reply:
ah i see